src/controllers/write/bugs.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const bugController = module.exports;
+const bugCreate = require('../../bugs');
+
+bugController.submitData = async function (req, res) {
+    bugCreate.post(req.body);
+    console.log(req.body);
+    res.render('Successful Bug Form Submit', req);
```

Comment:
Don't think rendering is necessary here since we are in a write controller. Should end with something like: 
`helpers.formatApiResponse(200, res);`
You can take a look at 'Topics.assignGroup' in controllers/write/topics.js for reference.

Reply:
Fixed